src/plopp/graphics/static.py

```diff
@@ -16,10 +27,16 @@ def _repr_mimebundle_(self, include=None, exclude=None) -> dict:
         """
         Mimebundle display representation for jupyter notebooks.
         """
-        return {
-            'text/plain': 'Figure',
-            'image/svg+xml': fig_to_bytes(self.canvas.fig, form='svg').decode()
-        }
+        out = {'text/plain': 'Figure'}
+        if self._repr_format is not None:
+            repr_maker = REPR_MAP[self._repr_format.lower()]
+        else:
+            npoints = 0
+            for line in self.canvas.ax.lines:
+                npoints += len(line.get_xdata())
+            repr_maker = REPR_MAP['png' if (npoints > 60_000) else 'svg']
```

Comment:
Why 60k specifically? Is it fast below that?

Reply:
Just trial and error on one machine. I don't have a good justification. Would probably be best to have something like "when the size of the svg output is more than the png bytes", but I don't really know how to predict that without first making both and comparing them, which seems counter-productive...

Reply:
60k SVG elements seems a lot. Have you considered, e.g., 10k?